### ISSUE TITLE
Allow to strip search params via config

### ISSUE THREAD

**Issue by fabian-hiller** [umami-software/umami#2529]
### Describe the feature or enhancement

It would be nice if we could strip search parameters via a config. This may improve privacy and fixes #2260 for me.

Example:

```html
<script
  defer
  src="http://mywebsite.com/umami.js"
  data-website-id="94db1cb1-74f4-4a40-ad6c-962362670409"
  data-strip-params="true"
/>
```

**Comment by mikecao** [comment]
We've implemented a different fix in v2.10.0 so that long params won't throw an error anymore.

**Comment by fabian-hiller** [comment]
What do you think about this as a configuration to improve privacy when needed? Search parameters often contain user-specific data.
